crates/store/re_types/definitions/rerun/components/fill_mode.fbs

```diff
@@ -4,13 +4,18 @@ namespace rerun.components;
 enum FillMode: byte(
     "attr.docs.unreleased"
 ) {
-    /// Lines are drawn around the edges of the shape.
+    /// Lines are drawn around the edges of the shape that represent the logged data.
     ///
-    /// The interior (2D) or surface (3D) are not drawn.
-    Wireframe (default),
+    /// The interior (2D) or surface (3D) are not filled in.
+    MajorWireframe (default),
+
+    /// Many lines are drawn to represent the surface of the shape.
+    ///
+    /// The interior (2D) or surface (3D) are not filled in.
```

Comment:
These docstrings doesn't tell me much. It would be more helpful with explicit examples of what these means for `Boxes3D` and `Ellipsoids3D`

Reply:
I have rewritten the documentation and included examples.